addons/mail/static/src/utils/common/format.js

```diff
@@ -274,4 +274,22 @@ export function parseEmail(text) {
     return [text, false];
 }
 
+/**
+ * Split the string using the splitChar and find all the valid/invalid emails from it.
+ * It return a object containing the list of all the emails and the invalid emails from it.
+ * Validity is handled by the parseEmail function.
+ *
+ * @param {string} splitPattern
+ * @param {string} emailStr
+ * @return {object}
+ */
+export function checkEmailValidity(splitPattern, emailStr){
```

Comment:
Hello, please move to `module` where it is used, as it appears to be of limited use as a genera tool.

Reply:
As the task has been split in two and the part requiring this MOV is no longer in this PR, I've removed the MOV here, but it will reappear in a future PR!